### PR TITLE
Add release note for v1.22.3

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,44 @@ This changelog goes through the changes that have been made in each release
...
+  * Fix the logic of finding the latest pull review commit ID (#32139) (#32165)
+  * Fix bug in getting merged pull request by commit (#32079) (#32117)
+  * Fix wrong last modify time (#32102) (#32104)
+  * Fix incorrect `/tokens` api (#32085) (#32092)
+  * Handle invalid target when creating releases using API (#31841) (#32043)
+  * Check if the `due_date` is nil when editing issues (#32035) (#32042)
+  * Fix container parallel upload bugs (#32022)
+  * Fixed race condition when deleting documents by repoId in ElasticSearch (#32185) (#32188)
+  * Increase `cacheContextLifetime` to reduce false reports (#32011) (#32023)
+  * Refactor CSRF protector (#32057) (#32069)
```
Comment (yp05327) [inline CHANGELOG.md:36]: performance or enhancement?

Comment (lunny) [reply CHANGELOG.md:36]: c3305cb4462e74930652c6d7c69a4942c5b629c2

**Comment by lunny** [comment]
wait backport of https://github.com/go-gitea/gitea/pull/32204

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,45 @@ This changelog goes through the changes that have been made in each release
...
 
+## [1.22.3](https://github.com/go-gitea/gitea/releases/tag/1.22.3) - 2024-10-06
+
+* SECURITY
+* PERFORMANCE
+  * Increase `cacheContextLifetime` to reduce false reports (#32011) (#32023)
+  * Don't join repository when loading action table data (#32127) (#32143)
+* BUGFIXES
+  * Fix javascript error when an anonymous user visiting migration page (#32144) (#32179)
+  * Don't init singing keys if oauth2 provider disabled (#32177)
```
Comment (sebastian-sauer) [inline CHANGELOG.md]:
```suggestion
  * Don't init signing keys if oauth2 provider is disabled (#32177)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,45 @@ This changelog goes through the changes that have been made in each release
...
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.22.3](https://github.com/go-gitea/gitea/releases/tag/1.22.3) - 2024-10-06
+
+* SECURITY
+* PERFORMANCE
+  * Increase `cacheContextLifetime` to reduce false reports (#32011) (#32023)
+  * Don't join repository when loading action table data (#32127) (#32143)
+* BUGFIXES
+  * Fix javascript error when an anonymous user visiting migration page (#32144) (#32179)
```
Comment (sebastian-sauer) [inline CHANGELOG.md]:
```suggestion
  * Fix javascript error when an anonymous user visits migration page (#32144) (#32179)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,45 @@ This changelog goes through the changes that have been made in each release
...
+  * Don't init singing keys if oauth2 provider disabled (#32177)
+  * Fix wrong status of `Set up Job` when first step is skipped (#32120) (#32125)
+  * Fix bug when deleting a migrated branch (#32075) (#32123)
+  * Truncate commit message during Discord webhook push events (#31970) (#32121)
+  * Allow set branch protection in an empty repository (#32095) (#32119)
```
Comment (sebastian-sauer) [inline CHANGELOG.md]:
```suggestion
  * Allow to set branch protection in an empty repository (#32095) (#32119)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,45 @@ This changelog goes through the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.22.3](https://github.com/go-gitea/gitea/releases/tag/1.22.3) - 2024-10-06
+
+* SECURITY
```
Comment (lafriks) [inline CHANGELOG.md:9]: This seems not to have items in it

Comment (lunny) [reply CHANGELOG.md:9]: c9e3022

**Review by lafriks — APPROVED** [review]

**Review by Zettat123 — APPROVED** [review]

**Pull request merged**